CONTRIBUTING.md

```diff
@@ -81,7 +81,7 @@ npm install
 
 First, get your free SendGrid account [here](https://sendgrid.com/free?source=sendgrid-nodejs).
 
-You will need to setup the following environment to use the SendGrid examples in the [README.md](https://github.com/sendgrid/sendgrid-nodejs/blob/master/README.md), [USAGE.md](https://github.com/sendgrid/sendgrid-nodejs/blob/master/USAGE.MD) and [USE_CASES.md](https://github.com/sendgrid/sendgrid-nodejs/blob/master/USE_CASES.md) files:
+You will need to setup the following environment to use the SendGrid examples in the [README.md](https://github.com/sendgrid/sendgrid-nodejs/blob/master/README.md), [USAGE.md](https://github.com/sendgrid/sendgrid-nodejs/blob/master/USAGE.md) and [USE_CASES](https://github.com/sendgrid/sendgrid-nodejs/blob/master/use-cases/README.md) files:
```

Comment:
I think we should use relative links in our docs to have GH link to the proper file based on whatever branch someone might be viewing from. https://blog.github.com/2013-01-31-relative-links-in-markup-files/

Also not sure about removing .md from the link, or maybe instead remove .md from all for consistency.